Successfully updated:
Created Frame node xewAIW2EI
Created Frame node mJYxbVg6r
Created Text node D2Vc2RMgX
Created Text node aQGjvSvx1
Created Frame node Ph1s9y8XC
Updated attributes for node xewAIW2EI
Updated attributes for node mJYxbVg6r
Updated text for node D2Vc2RMgX
Updated attributes for node D2Vc2RMgX
Updated text for node aQGjvSvx1
Updated attributes for node aQGjvSvx1
Updated attributes for node Ph1s9y8XC
Reordered node mJYxbVg6r within parent xewAIW2EI to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -288,41 +288,41 @@
               </AvatarWrap>
             </AvatarGroup>
           </BadgeGroup>
           <Heading
               nodeId="JPcat5fzD"
               width="1fr"
               height="fit-content"
               <!-- project text style path, always starts with / -->
               inlineTextStyle="/undefined - Test 95"
           >
             Fast PostgreSQL Search
           </Heading>
           <Description
               nodeId="yK6cCeTUB"
               width="1fr"
               height="fit-content"
               maxWidth="520px"
               <!-- project text style path, always starts with / -->
               inlineTextStyle="/Body lg"
           >
-            Updated text 8972
+            Updated text 4984
           </Description>
           <Stack
               nodeId="dxyO0shvx"
               width="100%"
               height="200px"
               <!-- background color string or project color style path (if starts with /) -->
               backgroundColor="rgb(50, 100, 200)"
               borderRadius="12px"
               layout="stack"
               gap="20px"
               padding="24px"
               stackDirection="horizontal"
               stackDistribution="center"
               stackAlignment="center"
           >
             <Frame
                 nodeId="i7j9JBlbE"
                 width="100px"
                 height="100px"
                 <!-- background color string or project color style path (if starts with /) -->
@@ -3898,20 +3898,73 @@
       width="fit-content"
       height="fit-content"
       top="0px"
       left="0px"
       font="Inter"
       textTruncation="2"
   >
     Truncated Text that is long enough to be truncated
   </TruncatedTextThatIsLongEnoughToBeTruncated>
 </Frame>
 <Frame
     nodeId="yKtdVfGM5"
     position="absolute"
     width="100px"
     height="100px"
     top="0px"
     left="0px"
     <!-- background color string or project color style path (if starts with /) -->
     backgroundColor="rgb(200, 200, 200)"
  />
+<Stack
+    nodeId="xewAIW2EI"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="mJYxbVg6r"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="D2Vc2RMgX"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="aQGjvSvx1"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="Ph1s9y8XC"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.